terraform/container_service.tf

```diff
@@ -10,13 +10,13 @@ data "template_file" "container_instance_cloud_config" {
 }
 
 module "container_service_cluster" {
-  source = "github.com/azavea/terraform-aws-ecs-cluster?ref=0.8.0"
+  source = "github.com/azavea/terraform-aws-ecs-cluster?ref=2.0.0"
```

Comment:
This upgrade introduces a few changes.

```
  - module.container_service_cluster.aws_iam_role.ecs_autoscale_role

  - module.container_service_cluster.aws_iam_role_policy_attachment.ecs_service_autoscaling_role
```
Autoscaling was removed in [2.0.0](https://github.com/azavea/terraform-aws-ecs-cluster/blob/develop/CHANGELOG.md#200) of the AWS ECS Cluster module.

```
  - module.container_service_cluster.aws_launch_configuration.container_instance

  + module.container_service_cluster.aws_launch_template.container_instance
    <truncated>
```
Launch configurations were replaced by launch templates in 2.0.0 as well.